### ISSUE TITLE
VAE loss

### ISSUE THREAD

**Issue by hongzimao** [ikostrikov/TensorFlow-VAE-GAN-DRAW#15]
Hi,

Awesome code! 

A minor issue. Following paper https://arxiv.org/pdf/1312.6114.pdf, equation 10, it seems line 62 of vae.py

`tf.reduce_sum(0.5 * (tf.square(mean) + tf.square(stddev) -
                                    2.0 * tf.log(stddev + epsilon) - 1.0))`

should be 

`tf.reduce_sum(0.5 * (tf.square(mean) + tf.square(stddev) -
                                     tf.log(stddev + epsilon) - 1.0))`
